test/it/client_it.js

```diff
@@ -16,19 +16,16 @@ describe('Client', function() {
   describe('creation', function() {
     it('should not throw', function(done) {
       assert.doesNotThrow(function () {
-        helpers.loadApiKey(function(apiKey) {
-          var client = new stormpath.Client({
-            apiKey: apiKey,
-            skipRemoteConfig: true
-          });
-
-          client.on('error', function (err) {
-            throw err;
-          });
-
-          client.on('ready', function () {
-            done();
-          });
+        var client = new stormpath.Client({
+          skipRemoteConfig: true
+        });
+
+        client.on('error', function (err) {
+          throw err;
+        });
+
+        client.on('ready', function () {
```

Comment:
If the `ready` event sends an error as the first argument, it should be passed on to the `done` callback:

```javascript
client.on('ready', done);
```

Or:

```javascript
client.on('ready', function (err) {
    done(err);
});
```